generators/client/templates/src/main/webapp/app/blocks/config/_compile.config.js

```diff
@@ -5,13 +5,15 @@
         .module('<%=angularAppName%>')
         .config(compileServiceConfig);
 
-    compileServiceConfig.$inject = ['$compileProvider', 'ENV'];
+    compileServiceConfig.$inject = ['$compileProvider'];
 
-    function compileServiceConfig($compileProvider, ENV) {
+    function compileServiceConfig($compileProvider) {
         // disable debug data on prod profile to improve performance
-        if(ENV === 'prod'){
-            $compileProvider.debugInfoEnabled(false);
-        }
+        
+        // TODO: Howto fetch the dynamic profile in this config.... can this be done elsewhere ?
```

Comment:
@deepu105 :  I did look at that, but I assumed providers allowed you to "configure" services at config time, before they become actual services that you inject.

In this case, we would need to do a service invocation (to get the profile info) in the config. I felt it would "hack" the provider mechanism to actually invoke the service itself. Also, I don't think you can even access $resource, $q outside of the get function of the provider. And in this proposal, we would need it outside the get function.

An alternative could be to just set the debugInfo flag statically during build time (using gulp like the ENV was done). 

I still think there's a big different between the deugInfo flag (more static in nature) and the actual environment (more dynamic).